php/functions.php

```diff
@@ -190,7 +190,8 @@ function parsePeers($peers, $curl_handle)
     foreach ($peers as $peer) {
 
         // Extract IP address for later
-        $peer_addr_array = explode(':', $peer['addr']);
+        //$peer_addr_array = explode(':', $peer['addr']);
```

Comment:
Can you fully remove this line, rather than commenting it out? If needed, we can always revert the commit.